packages/ui/Skinnable/tests/Skinnable-specs.js

```diff
@@ -360,56 +362,17 @@ describe('Skinnable Specs', () => {
 		const SkinnableParent = Skinnable(config, Component);
 		const SkinnableChild = Skinnable(config, Component);
 
-		const subject = mount(
+		render(
 			<SkinnableParent>
 				<SkinnableChild skinVariants={{normal: false}}>
-					<SkinnableChild id="innerChild" />
+					<SkinnableChild data-testid="innerChild" />
 				</SkinnableChild>
 			</SkinnableParent>
 		);
 
 		const expected = 'darkSkin';
-		const actual = subject.find('div#innerChild').prop('className');
-
-		expect(actual).toEqual(expected);
-	});
-
-	test('should not force re-render of child if child unaffected', () => {
```

Comment:
🤔 I found this test is failing now. Let's not remove this test and just skip it for now. I'll create a following ticket.